packages/cadl-ranch-expect/package.json

```diff
@@ -35,9 +35,9 @@
     "typescript": "^5.0.4"
   },
   "peerDependencies": {
-    "@typespec/compiler": "~0.43.0",
-    "@typespec/versioning": "~0.43.0",
-    "@typespec/http": "~0.43.1",
-    "@typespec/rest": "~0.43.0"
+    "@typespec/compiler": "0.44.0-dev.21",
```

Comment:
do we need to bump all of these packages to get encode?

Reply:
I would just not merge this until tomorrow, the typesepc release will be there and can just use the next stable

Reply:
After the tsp next version released, I'll resolve the dependencies.